src/helpers/tests/measureValueFilterOperator.spec.ts

```diff
@@ -0,0 +1,23 @@
+// (C) 2007-2019 GoodData Corporation
+import * as Operators from "../../constants/measureValueFilterOperators";
+import { getOperatorTranslationKey, getOperatorIcon } from "../measureValueFilterOperator";
+
+describe("getOperatorTranslationKey", () => {
+    it("should return valid translation key", () => {
```

Comment:
This should be 4 separate tests, IMO (or even better a parametrized test). The reason for that is that if you have this "big" test, you won' immediately know what is wrong when it fails - did `ALL` fail, or `BETWEEN`?
The same applies to the test in the next `describe`.